doc/specs/#4993 - Keyboard Selection/Keyboard-Selection.md

```diff
@@ -110,10 +117,65 @@ For `SelectionExpansion = Buffer`, the selection endpoint will be moved to the b
 
 **NOTE**: In all cases, horizontal movements attempting to move past the left/right viewport boundaries result in a wrap. Vertical movements attempting to move past the top/bottom viewport boundaries will scroll such that the selection is at the edge of the screen. Vertical movements attempting to move past the top/bottom buffer boundaries will be clamped to be within buffer boundaries.
 
-Every combination of the `SelectionDirection` and `SelectionExpansion` will map to a keybinding. These pairings are shown below in the UI/UX Design --> Keybindings section.
-
 **NOTE**: If `copyOnSelect` is enabled, we need to make sure we **DO NOT** update the clipboard on every change in selection. The user must explicitly choose to copy the selected text from the buffer.
 
+### Copy Mode
+Copy Mode is a mode where the user can create and modify a selection using only the keyboard. The following flowchart covers how the new `copymode()` action works:
+
+![Copy Mode Flowchart][images/CopyModeFlowchart.png]
+
+**NOTE**: `copyMode()` refers to the action, whereas `updateSelection()` refers to the underlying function that is being called in the code.
+
+If a selection is not active, a "start" and "end" selection point is created at the cursor position. `updateSelection()` calls then move "start" and "end" together as one position. This position will be denoted with a "y-beam".

```

Comment:
okay so this does kill my "`shift+arrow` to enter mark mode and then move the endpoint like conhost" dream. According to this, we'll start moving both, and I'd need to press another key to just move one of them.

Reply:
Yeah :/. A part of the problem is that <kbd>shift+arrow</kbd> is used by PowerShell too. So I feel like we don't want a default binding on them (the non-configurable keyboard selection ones are kinda an exception because they're conditionally applied).

On a more positive note, `copyMode()` would appear in the command palette, which I feel is nice.

Reply:
Oh I'm certainly not suggesting making that a default, just something I (as a cmd user) want :P